README.md

```diff
@@ -102,6 +102,14 @@ A sample configuration file:
 }
 ```
 
+```toml
```

Comment:
I don't think TOML is common enough to get a dedicated section - especially when YAML doesn't. What about removing this and adding a link to one of the TOML test files on the following line that links to examples?

Reply:
I see your point and removed the section.
Linking to the test example is not the best solution since the other examples given are way more complete. I converted the "base-config.json" file to toml and added it to the test folder to have an example to link.